src/components/Filter/filter.css

```diff
@@ -41,3 +40,15 @@
   font-size: 0.8rem;
   color: var(--dark-primary-clr);
 }
+
+@media screen and (max-width: 600px) {
+  .productlisting_sidebar {
+    width: 40%;
+  }
+}
+
+@media screen and (max-width: 450px) {
+  .price-range-inp {
```

Comment:
For most of the selectors you have used underscore`_`, but here it's hyphen`-`, so that should be consistent.

Reply:
Yaah I will change those

Reply:
In next PR since I will have to change component library for that